variables.tf

```diff
@@ -50,7 +50,7 @@ variable "executor_machine_image" {
 
 variable "executor_machine_type" {
   type        = string
-  default     = "n1-standard-4" // 4 vCPU, 15GB
+  default     = "c2-standard-8" // 8 vCPU, 32GB
```

Comment:
this is the default we use everywhere, so adjusted it in here, too. Interestingly, c2 instances are way cheaper in spot pricing.